cns/requestcontroller/kubecontroller/crdrequestcontroller.go

```diff
@@ -32,10 +33,12 @@ const (
 	prometheusAddress = "0" //0 means disabled
 )
 
-// crdRequestController
+var _ requestcontroller.RequestController = (*requestController)(nil)
+
+// requestController
 // - watches CRD status changes
 // - updates CRD spec
-type crdRequestController struct {
```

Comment:
crdRequestController will make it specific to single tenant crd scenarios...for multi tenant crds, we are using multitenantCrdRequestController

Reply:
since the multitenant request controller is in a different package, it's unambiguous and having the name be simpler is better - the package is part of the name when it is imported and used so no information is lost

Reply:
Agreed with Matt above, if we are fixing it, then lets fix it completely. Either we rename this file to SingleTenantRequestController and similarly the structure or what Matt suggested above